Lib/test/test_poll.py

```diff
@@ -204,6 +204,39 @@ def test_threaded_poll(self):
             os.write(w, b'spam')
             t.join()
 
+    @unittest.skipUnless(threading, 'Threading required for this test.')
+    @reap_threads
+    def test_poll_blocks_with_negative_ms(self):
+
+        for timeout_ms in [None, -1, -1.0, -0.1, -1e-100]:
+
+            # GIVEN
```

Comment:
Sorry, but I don't understand what mean these GIVEN/WHEN/THEN.

Reply:
Sorry, I add then for clarity when testing ([example](https://solidsoft.wordpress.com/2017/05/16/importance-of-given-when-then-in-unit-tests-and-tdd/)).

I will delete them if you want.

Reply:
Done in 5c58695